specs.json

```
@@ -190,6 +190,30 @@
   "https://privacycg.github.io/requestStorageAccessFor/",
   "https://privacycg.github.io/storage-access/",
   "https://quirks.spec.whatwg.org/",
+  {
+    "url": "https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html",
+    "shortname": "glTF",
+    "groups": [
+      {
+        "name": "glTF Working Group",
+        "url": "https://www.khronos.org/gltf/"
+      }
+    ],
+    "nightly": {
+      "sourcePath": "specification/2.0/Specification.adoc"
+    }
```

```suggestion
    },
    "categories": [
      "-browser"
    ]
```
(since it's not implemented in browsers at the moment)